tools/packaging/tempo-postinstall.sh

```diff
@@ -16,6 +16,10 @@ cleanInstall() {
         adduser --system --shell /bin/false "tempo"
     fi
 
+    # Create and assign permissions for default storage
+    mkdir /var/tempo
```

Comment:
maybe `mkdir -m 0700 /var/tempo` so that non-privileged users can't read block data, etc.